app/main.py

```diff
@@ -1,8 +1,43 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name, age):
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    peoples_dict = {}
+    for people_one in people:
+        name = people_one["name"]
+        age = people_one["age"]
+        peoples_dict[f"{name}"] = Person(name, age)
+
+        try:
+            wife = people_one["wife"]
+            peoples_dict[f"{name}"].wife = wife
+        except Exception:
+            husband = people_one["husband"]
+            peoples_dict[f"{name}"].husband = husband
+
+    for key in list(peoples_dict.keys()):
+        try:
```

Comment:
Why do you use try here? 
Use if instead. You can check if the attribute `wife` or `husband` exists using `hasatttr`